### ISSUE TITLE
Optional Border

### ISSUE THREAD

**Issue by GoogleCodeExporter** [thaindq/opensudoku-android#7]
```
Since it can sometimes be hard to click on cells near the edge of the 
screen (I assume because of the way the touch screen works), it would be 
nice to be able to have a border around the board.

Great work so far!
```

Original issue reported on code.google.com by `mogulgu...@gmail.com` on 31 May 2009 at 5:06

**Comment by GoogleCodeExporter** [comment]
Original comment by `romario...@gmail.com` on 3 Jun 2009 at 7:59

**Comment by GoogleCodeExporter** [comment]
Original comment by `romario...@gmail.com` on 7 Sep 2009 at 5:35

**Comment by GoogleCodeExporter** [comment]
Original comment by `romario...@gmail.com` on 29 Sep 2009 at 1:28
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
I've added option "Screen border size" (in Game Settings/Game/More 
Settings/Screen
border size).

Issue is solved in 0.7.1, which has not been released yet. If you want to test 
it
before official release, you can download it here:
http://code.google.com/p/opensudoku-android/downloads/list (look for OpenSudoku 
0.7.1
- unstable). Please note that if you install this test version, you will 
probably no
longer receive updates from market, so when you're finished with testing, 
install
OpenSudoku from market again.
```

Original comment by `romario...@gmail.com` on 29 Sep 2009 at 2:48

**Comment by GoogleCodeExporter** [comment]
Original comment by `romario...@gmail.com` on 2 Oct 2009 at 5:45
- Changed state: **Fixed**
- Added labels: **Milestone-0.7.1**
